app/models/tasks/copy_media_task.rb

```diff
@@ -48,8 +48,13 @@ def update_media_resource
         media_resource.duration = porter_callback_transcode[:Duration]&.to_f&./ 1000
       end
 
-      # change status, if metadata doesn't pass validations
-      media_resource.status = "invalid" if media_resource.invalid?
+      # change status, for invalid and bad-duration files
+      if media_resource.invalid?
+        media_resource.status = "invalid"
+      elsif info[:Audio].try(:[], :DurationDiscrepancy).to_i > 500
```

Comment:
Is there any case this could accidentally get called on existing audio?
Older files won't have a DurationDiscepancy, so it would just be a 0, which is less than 500, so that's ok.

Seems like it should be fine, just worrying about any unnecessary processing of old files or worse processing that might overwrite files and cause an issue...